src/protocols/core/Output.cpp

```diff
@@ -55,6 +53,8 @@ void CWLOutputResource::updateState() {
     if (resource->version() >= 2)
         resource->sendScale(std::ceil(monitor->scale));
 
+    resource->sendGeometry(monitor->vecPosition.x, monitor->vecPosition.y, monitor->output->physicalSize.x, monitor->output->physicalSize.y, (wl_output_subpixel)monitor->output->subpixel, monitor->output->make.c_str(),
```

Comment:
why this change?

Reply:
This event should be sent when any of the properties change. But without this change this is not done.
In EIS or Input Capture portal, i need to notify whenever the output geometry change.
[wl_output::geometry](https://wayland.app/protocols/wayland#wl_output:event:geometry)

Reply:
ok